applications/browser/package.json

```diff
@@ -0,0 +1,124 @@
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eclipse-theia/theia-blueprint.git"
+  },
+  "engines": {
+    "yarn": ">=1.7.0 <2",
+    "node": ">=12.14.1"
+  },
+  "theia": {
+    "frontend": {
+      "config": {
+        "applicationName": "Theia Blueprint",
+        "preferences": {
+          "toolbar.showToolbar": true,
+          "files.enableTrash": false
+        }
+      }
+    },
+    "backend": {
+      "config": {
+        "startupTimeout": -1,
+        "resolveSystemPlugins": false
+      }
+    },
+    "generator": {
+      "config": {
+        "preloadTemplate": "./resources/preload.html"
+      }
+    }
+  },
+  "dependencies": {
+    "@theia/bulk-edit": "1.35.0",
+    "@theia/callhierarchy": "1.35.0",
+    "@theia/console": "1.35.0",
+    "@theia/core": "1.35.0",
+    "@theia/debug": "1.35.0",
+    "@theia/editor": "1.35.0",
+    "@theia/editor-preview": "1.35.0",
+    "@theia/external-terminal": "1.35.0",
+    "@theia/file-search": "1.35.0",
+    "@theia/filesystem": "1.35.0",
+    "@theia/getting-started": "1.35.0",
+    "@theia/git": "1.35.0",
+    "@theia/keymaps": "1.35.0",
+    "@theia/markers": "1.35.0",
+    "@theia/messages": "1.35.0",
+    "@theia/memory-inspector": "1.35.0",
+    "@theia/metrics": "1.35.0",
+    "@theia/mini-browser": "1.35.0",
+    "@theia/monaco": "1.35.0",
+    "@theia/navigator": "1.35.0",
+    "@theia/outline-view": "1.35.0",
+    "@theia/output": "1.35.0",
+    "@theia/plugin-dev": "1.35.0",
+    "@theia/plugin-ext": "1.35.0",
+    "@theia/plugin-ext-vscode": "1.35.0",
+    "@theia/preferences": "1.35.0",
+    "@theia/process": "1.35.0",
+    "@theia/property-view": "1.35.0",
+    "@theia/scm": "1.35.0",
+    "@theia/scm-extra": "1.35.0",
+    "@theia/search-in-workspace": "1.35.0",
+    "@theia/task": "1.35.0",
+    "@theia/terminal": "1.35.0",
+    "@theia/timeline": "1.35.0",
+    "@theia/toolbar": "1.35.0",
+    "@theia/typehierarchy": "1.35.0",
+    "@theia/userstorage": "1.35.0",
+    "@theia/variable-resolver": "1.35.0",
+    "@theia/vsx-registry": "1.35.0",
+    "@theia/workspace": "1.35.0",
+    "fs-extra": "^9.0.1",
+    "theia-blueprint-updater": "1.35.0",
+    "theia-blueprint-product": "1.35.0"
+  },
+  "devDependencies": {
+    "@theia/cli": "1.35.0"
+  },
+  "scripts": {
+    "prepare": "yarn build && yarn download:plugins",
+    "clean": "theia clean && rimraf node_modules",
+    "build": "yarn -s compile && yarn -s bundle",
+    "bundle": "theia build",
+    "compile": "tsc -b",
+    "rebuild": "theia rebuild:browser --cacheRoot ../..",
+    "start": "yarn -s rebuild && theia start --plugins=local-dir:./plugins",
+    "watch": "concurrently --kill-others -n tsc,bundle -c red,yellow \"tsc -b -w --preserveWatchOutput\" \"yarn -s watch:bundle\"",
+    "download:plugins": "theia download:plugins"
+  },
+  "theiaPluginsDir": "plugins",
+  "theiaPlugins": {
+    "vscode.git": "https://open-vsx.org/api/vscode/git/1.52.1/file/vscode.git-1.52.1.vsix",
```

Comment:
I think this version (1.52.1) probably works correctly (i.e. not affected by the bug mentioned elsewhere on this page). 

I agree it would be clearer to remove the extension from the exclude list, but that list I think affects only extensions that are part of a pack, and so should have no effect on an extension mentioned explicitly in `theiaPlugins`.

Reply:
I've aligned the plugin lists while keeping `vscode.git` in the browser application. In te browser app the excludes were adjusted accordingly

Reply:
@jfaltermeier I've been told that the issue [1] with the "vscode builtin git" seem to happens even with older versions. If you can confirm, we might want to switch to the Theia-specific git extension, until the problem is fixed. 

[1] https://github.com/eclipse-theia/theia/issues/12173

Reply:
Ah, yes I can confirm it also happens with the older version. I didn't notice before, because I still had `@theia/git` in the plugins list as well and both contributed their toolbar items. I've excluded `vscode.git` and `vscode.git-ui` again